buildSrc/src/main/java/io/micronaut/guides/core/GuidesConfigurationProperties.java

```diff
@@ -31,6 +34,9 @@ public class GuidesConfigurationProperties implements GuidesConfiguration {
     private String envJdkVersion = DEFAULT_ENV_JDK_VERSION;
     private JdkVersion defaulJdkVersion = DEFAULT_JAVA_VERSION;
     private List<JdkVersion> jdkVersionsSupportedByGraalvm = DEFAULT_JDK_VERSIONS_SUPPORTED_BY_GRAALVM;
+    private String githubWorkflowJavaCi = GITHUB_WORKFLOW_JAVA_CI;
```

Comment:
I don't see when a user is going to want to customize ` githubWorkflowJavaCi` or `envGithubWorkflow`